.github/workflow/make_unique.yml

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
this workflow does not work, because it's in workflow dir instead of workflow**s**